src/utils/constants.js

```diff
@@ -43,6 +46,7 @@ export const cardListEl = document.querySelector(".cards__list");
 export const userInfo = new UserInfo({
```

Comment:
* You haven't corrected my remark https://github.com/Jazzmine-Flora/se_project_aroundtheus/pull/1#discussion_r1730151351  Instances of classes are not **constants**. Please, move them into `index.js`


```js

// Create instances of FormValidator for each form
export const profileFormValidator = new FormValidator(
  validationConfig,
  profileForm
);
export const newCardFormValidator = new FormValidator(
  validationConfig,
  newCardForm
);

export const userInfo = new UserInfo({
  nameSelector: ".profile__title",
  jobSelector: ".profile__description",
  avatarSelector: ".profile__image",
});

```


These instances should be in `index.js`. Please, remove them from `constants.js`